README.md

```diff
@@ -5,9 +5,10 @@
 ```sh
 cd ~
 
-git clone https://github.com/Lokeh/plum && mv plum .plum
+git clone https://github.com/Lokeh/plum .plum
 
-cp .plum/plum /usr/local/bin/plum
+# customize this location to be somewhere on your $PATH
+ln -s .plum/plum /usr/local/bin/plum
```

Comment:
On Ubuntu, `ln -s .plum/plum /usr/local/bin/plum` gives

> ln: failed to create symbolic link '/usr/local/bin/plum': Permission denied

This might not be the case on Mac, that's not my table of expertice. On Ubuntu, I'd have to

    sudo ln -s .plum/plum /usr/local/bin/plum

to achieve a global installation. Optionally, without super user permissions, I could

    ln -s .plum/plum ~/.local/bin/plum

or

    ln -s .plum/plum ~/bin/plum

depending on whether I use ~/.local/bin or ~/bin for scripts. Both are quite normal.

---

Personal recommendation: 

```suggestion
Install globally with

    ln -s ~/.plum/plum /usr/local/bin/plum
    
, or link to another folder on your path as `~/bin`.
```